patacrep/content/song.py

```diff
@@ -85,20 +90,24 @@ def parse(keyword, argument, contentlist, config):
                     LOGGER.debug('Parsing file "{}"…'.format(filename))
                     extension = filename.split(".")[-1]
                     if extension not in plugins:
-                        LOGGER.warning(
+                        LOGGER.info(
                             (
-                                'I do not know how to parse "{}": name does '
-                                'not end with one of {}. Ignored.'
-                            ).format(
-                                os.path.join(songdir.datadir, filename),
-                                ", ".join(["'.{}'".format(key) for key in plugins.keys()]),
-                                ))
+                                'I do not know how to parse "%s": name does '
```

Comment:
C'est bizarre le `I do not know...`, de mémoire on utilise plus des formulations du type `Can't parse...` (mais je pinaille ;-)